frontend/src/concepts/areas/const.ts

```diff
@@ -115,6 +116,12 @@ export const SupportedAreasStateMap: SupportedAreasState = {
     requiredComponents: [StackComponent.MODEL_REGISTRY],
     requiredCapabilities: [StackCapability.SERVICE_MESH, StackCapability.SERVICE_MESH_AUTHZ],
   },
+  [SupportedArea.SERVING_RUNTIME_PARAMS]: {
+    featureFlags: ['disableServingRuntimeParams'],
+    requiredComponents: [StackComponent.K_SERVE],
+    reliantAreas: [SupportedArea.MODEL_SERVING],
+    requiredCapabilities: [StackCapability.SERVICE_MESH, StackCapability.SERVICE_MESH_AUTHZ],
```

Comment:
I'm not sure this does actually rely on servicemesh.. is that something I missed in refinement? If we're allowing Kserve model deployment we should be able to support these params. I think this area has the same requirements as `disableKServeMetrics`, so it can probably just have `reliantAreas: [SupportedArea.K_SERVE, SupportedArea.MODEL_SERVING],` and no `requiredComponents` or `requiredCapabilities`

Reply:
Oh yes, I forgot to comment... I needed help with the changes in this file. Thanks. This is addressed.

Force-pushed coz there was a typo in the commit message.